.github/workflows/statitc-coda-analysis.yml

```diff
@@ -0,0 +1,21 @@
+name: static-code-analysis
```

Comment:
Typo in the file name. Also, maybe rename it to "code formatting"? We don't really do static analysis here (no mypy, no linters). Or do we intend to add these later?

Reply:
awww thanks :)

and yeah, I'd say that we should slowly introduce more static checking!